addons/apecloud-mysql/templates/cmpdversion.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentVersion
+metadata:
+  name: apecloud-mysql
+  labels:
+    {{- include "apecloud-mysql.labels" . | nindent 4 }}
+spec:
+  releases:
+  - name: apecloud-mysql-8.0.30
+    serviceVersion: "8.0.30"
+    images:
+      mysql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
+      mysql-exporter: {{ .Values.metrics.image.registry | default .Values.image.registry }}/{{ .Values.metrics.image.repository }}:{{ default .Values.metrics.image.tag }}
+      vtablet: {{ .Values.wesqlscale.image.registry | default "docker.io" }}/{{ .Values.wesqlscale.image.repository }}:{{ .Values.wesqlscale.image.tag }}
+  - name: apecloud-mysql-auditlog-8.0.30
+    serviceVersion: "8.0.30"
+    images:
+      init-data: {{ .Values.image.registry | default "docker.io" }}/apecloud/mysql_audit_log:8.0.33
+      mysql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
+      mysql-exporter: {{ .Values.metrics.image.registry | default .Values.image.registry }}/{{ .Values.metrics.image.repository }}:{{ default .Values.metrics.image.tag }}
+      vtablet: {{ .Values.wesqlscale.image.registry | default "docker.io" }}/{{ .Values.wesqlscale.image.repository }}:{{ .Values.wesqlscale.image.tag }}
+  compatibilityRules:
+  - releases: [apecloud-mysql-8.0.30]
+    compDefs: [apecloud-mysql]
+  - releases: [apecloud-mysql-auditlog-8.0.30]
+    compDefs: [apecloud-mysql-auditlog]
+---
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentVersion
+metadata:
+  name: wescale-vtcontroller
```

Comment:
It seems that it's good enough to provide a component version for wescale.

Reply:
there are two components for wescale,  vectonroller and vtgate, so use two cmpv, one for each.

Reply:
![Uploading image.png…]()

Reply:
fixed